maths/difference_sort.py

```diff
@@ -0,0 +1,34 @@
+def diff(a=int, c=int) -> int:
```

Comment:
```suggestion
def diff(b: list, d: list) -> list:
```
we must take input as parameters and the type hinting must be like this